tests/e2e/system_test/test_cr_resources_validation.py

```diff
@@ -14,55 +18,121 @@
 ERRMSG = "Error in command"
 
 
-@system_test
+@tier3
 @skipif_ocp_version("<4.13")
 @skipif_ocs_version("<4.13")
 class TestCRRsourcesValidation(E2ETest):
     """
     Test that check that csi addons resources are not editable after creation
     """
 
-    def test_network_fence_not_editable(self):
-        """
-        Test case to check that network fence object is not editable once created
-        """
+    def setup(self):
         self.temp_files_list = []
         self.object_name_to_delete = ""
-        network_fence_yaml = os.path.join(
-            constants.TEMPLATE_CSI_ADDONS_DIR, "NetworkFence.yaml"
-        )
-        res = run_oc_command(cmd=f"create -f {network_fence_yaml}")
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        """
+        Cleanup the test environment
+
+        """
+
+        def finalizer():
+            for temp_file in self.temp_files_list:
+                if os.path.exists(temp_file):
+                    run_cmd(f"rm {temp_file}")
+
+            if self.object_name_to_delete != "":
+                res = run_oc_command(cmd=f"delete {self.object_name_to_delete}")
+                assert (
+                    ERRMSG not in res[0]
+                ), f"Failed to delete network fence resource with name: {self.object_name_to_delete}, got result: {res}"
```

Comment:
Is this always network fence resource?

Reply:
Done in https://github.com/red-hat-storage/ocs-ci/pull/7792